Sources/PointFree/Routers/AccountRoutes.swift

```diff
@@ -0,0 +1,151 @@
+import ApplicativeRouter
+import ApplicativeRouterHttpPipelineSupport
+import Either
+import Foundation
+import HttpPipeline
+import Optics
+import Prelude
+import Styleguide
+import Tuple
+
+public let accountRouter = accountRouters.reduce(.empty, <|>)
+
+extension Route {
+  public enum Account: DerivePartialIsos {
+    case confirmEmailChange(userId: Database.User.Id, emailAddress: EmailAddress)
+    case index
+    case invoices(Invoices)
+    case paymentInfo(PaymentInfo)
+    case rss(userId: Database.User.Id, rssSalt: Database.User.RssSalt)
+    case subscription(Subscription)
+    case update(ProfileData?)
+
+    public enum Invoices: DerivePartialIsos {
+      case index
+      case show(Stripe.Invoice.Id)
+    }
+
+    public enum PaymentInfo: DerivePartialIsos {
+      case show(expand: Bool?)
+      case update(Stripe.Token.Id?)
+    }
+
+    public enum Subscription: DerivePartialIsos {
+      case cancel
+      case change(Change)
+      case reactivate
+
+      public enum Change: DerivePartialIsos {
+        case show
+        case update(Pricing?)
+      }
+    }
+  }
+}
+
+private let accountRouters: [Router<Route.Account>] = [
+
+  .confirmEmailChange
+    <¢> get %> lit("confirm-email-change")
+    %> queryParam("payload", .appDecrypted >>> payload(.uuid >>> .tagged, .tagged))
+    <% end,
+
+  .index
+    <¢> get <% end,
+
+  .invoices <<< .index
+    <¢> get %> lit("invoices") <% end,
+
+  .invoices <<< .show
+    <¢> get %> lit("invoices") %> pathParam(.string >>> .tagged) <% end,
+
+  .paymentInfo <<< .show
+    <¢> get %> lit("payment-info")
+    %> queryParam("expand", opt(.bool))
+    <% end,
+
+  .paymentInfo <<< .update
+    <¢> post %> lit("payment-info")
+    %> formField("token", Optional.iso.some >>> opt(.string >>> .tagged))
+    <% end,
+
+  .rss
+    <¢> (get <|> head) %> lit("rss")
+    %> pathParam(.appDecrypted >>> .uuid >>> .tagged)
+    <%> pathParam(.appDecrypted >>> .uuid >>> .tagged)
```

Comment:
the user id and salt are encrypted and exposed in the path params of the private url.